adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -300,45 +270,368 @@ private[adam] class VariantContextConverter(
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttributeAsString("AA", null))
+      .fold(vab)(vab.setAncestralAllele(_))
+  }
+
+  private[converters] def formatDbSnp(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttribute("DB").asInstanceOf[java.lang.Boolean])
+      .fold(vab)(vab.setDbSnp(_))
+  }
+
+  private[converters] def formatHapMap2(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttribute("H2").asInstanceOf[java.lang.Boolean])
+      .fold(vab)(vab.setHapMap2(_))
+  }
+
+  private[converters] def formatHapMap3(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttribute("H3").asInstanceOf[java.lang.Boolean])
+      .fold(vab)(vab.setHapMap3(_))
+  }
+
+  private[converters] def formatValidated(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttribute("VALIDATED").asInstanceOf[java.lang.Boolean])
+      .fold(vab)(vab.setValidated(_))
+  }
+
+  private[converters] def formatThousandGenomes(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttribute("1000G").asInstanceOf[java.lang.Boolean])
+      .fold(vab)(vab.setThousandGenomes(_))
+  }
+
+  private[converters] def formatSomatic(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    Option(vc.getAttribute("SOMATIC").asInstanceOf[java.lang.Boolean])
+      .fold(vab)(vab.setSomatic(_))
+  }
+
+  private[converters] def formatAlleleCount(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    val ac = vc.getAttributeAsList("AC")
+    if (ac.size > index) {
+      vab.setAlleleCount(toInt(ac.get(index)))
+    }
+    vab
+  }
+
+  private[converters] def formatAlleleFrequency(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    val af = vc.getAttributeAsList("AF")
+    if (af.size > index) {
+      vab.setAlleleFrequency(toFloat(af.get(index)))
+    }
+    vab
+  }
+
+  private[converters] def formatCigar(
+    vc: HtsjdkVariantContext,
+    vab: VariantAnnotation.Builder,
+    v: Variant,
+    index: Int): VariantAnnotation.Builder = {
+
+    val cigar = vc.getAttributeAsList("CIGAR")
+    if (cigar.size > index) {
+      vab.setCigar(asString(cigar.get(index)))
+    }
+    vab
+  }
+
+  private[converters] def formatReadDepth(
```

Comment:
I would prefer this function to be `formatAllelicDepth`. When I hear "read depth", I think "DP", not "AD".